apps/settings/js/panels/hotspot_wifi_settings/panel.js

```diff
@@ -135,7 +134,8 @@ define(function(require) {
 
         // ensure SSID is set
         if (/^\s*$/.test(elements.tethering_ssid.value)) {
-          alert(_('SSIDCannotBeEmpty'));
+          navigator.mozL10n.formatValue('SSIDCannotBeEmpty').then(
+            msg => alert(msg));
```

Comment:
Disabling the OK button is a better solution. I'm surprised the design didn't go to this way. :-/